V6_MIGRATION.md

```diff
@@ -76,9 +76,9 @@ venmoClient.tokenize(venmoRequest) { venmoAccountNonce, error in
 
 The property `BTPayPalRequest.activeWindow` has been removed
 
-Removed `BTPayPalDriver.requestOneTimePayment` and `BTPayPalDriver.requestBillingAgreement` in favor of `BTPayPalClient.tokenizePayPalAccount`:
+Removed `BTPayPalDriver.requestOneTimePayment` and `BTPayPalDriver.requestBillingAgreement` in favor of `BTPayPalClient.tokenize`:
 ```
-let payPalClient = BTPayPalClient(apiClient: <MY_BTAPICLIENT>)
+let payPalClient = BTPayPalClient(apiClient: <MY_BTAPIClient>)
```

Comment:
Updated this but not totally sure what our preferred syntax is for this. All caps? Just `<BTAPIClient>`? Something else entirely? Hoping someone has a strong preference and we can update these all accordingly.

Reply:
Could we instantiate a BTAPIClient with a "<TOKENIZATION_KEY_OR_CLIENT_TOKEN>" auth string? Not sure if that's too much but it'll have a similar effect as above and be valid for the copy/paste.

Reply:
I think in general the `<SOME_VALUE>` syntax is all-caps, and is usually for strings.

I would do something like:
```
let apiClient = BTAPIClient("<TOKENIZATION_KEY_OR_CLIENT_TOKEN>")
let payPalClient = BTPayPalClient(apiClient: apiClient)
```

Reply:
And then in the examples below, just use `apiClient` as the param.

Reply:
Dope! I'll update these - I dig that too.

Reply:
Done: https://github.com/braintree/braintree_ios/pull/984/commits/5842ae352f8324be34cc5c372319765e22a2aa72